source/edge-server.txt

```diff
@@ -0,0 +1,126 @@
+:template: product-landing
+:hidefeedback: header
+:noprevnext:
+
+.. _edge-server:
+
+=================
+Atlas Edge Server
+=================
+
+.. toctree::
+   :titlesonly:
+   :hidden:
+   
+   Configure Edge Server </edge-server/configure>
+   Connect to Edge Server </edge-server/connect>
+   Use MongoDB APIs with Edge Server </edge-server/mongodb>
+
+.. introduction::
+
+   You have a warehouse, retail store, air-gapped laboratory, or another 
+   premises where devices should be able to seamlessly sync their data.
+   Those locations may or may not be online at any given time. Lack of a 
```

Comment:
maybe replace "online" with "connected to the internet"?

Reply:
The landing page is intended to be a quick overview of the functionality, and it's more marketing-focused. Using "online" here makes for more concise copy that flows easily. Do you feel "connect to the internet" is an important clarification to make in this context? I can change it if it's important to communicate that, but it won't be as snappy.